### PR TITLE
Vendor map

### PR DESCRIPTION
**Pull request by Zairbekov**

### CODE REVIEW

**Review by AlexanderBazukevich — APPROVED** [review]

**Review by abarmina — APPROVED** [review]

File: src/app/shared/custom-components/map/map.component.ts [hunk 1]
```diff
@@ -29,86 +30,81 @@ export class MapComponent implements OnInit, OnDestroy {
...
-        }
-      ),
-      latlng = L.latLng(0, 0);
-    this.map = L.map('map', { center: latlng, zoom: 11, layers: [tiles] });
-  }
-
-  private initOffersMarkers(offers: Offer[], city: string) {
-    this.mapService.getCityView(city).then((data) => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private initMarkers(data: any) {
```
Comment (tlazarchuk) [inline src/app/shared/custom-components/map/map.component.ts:83]: "data: any" is not the best decision

File: src/app/shared/custom-components/map/map.component.ts [hunk 2]
```diff
@@ -29,86 +30,81 @@ export class MapComponent implements OnInit, OnDestroy {
...
+    const { name, offices, vendorName } = data;
```
Comment (tlazarchuk) [inline src/app/shared/custom-components/map/map.component.ts:84]: You should create an Object value and give data to it

**Review by tlazarchuk — APPROVED** [review]

File: src/app/shared/custom-components/map/map.component.ts (hunk 1 above)
Comment (Zairbekov) [reply src/app/shared/custom-components/map/map.component.ts:83]: Run the code and try to understand why we used "any"

File: src/app/shared/custom-components/map/map.component.ts (hunk 2 above)
Comment (Zairbekov) [reply src/app/shared/custom-components/map/map.component.ts:84]: I don't understand clearly. if you mean about 84 line then it new method to take necessary values from Data.

**Pull request merged**